### PR TITLE
ForceScheduler: make errors more helpful by telling which scheduler has the issue

### PR DESCRIPTION
**Pull request by jaredgrubb**
I recently had a ForceScheduler that wasnt configured right, and the config error was not helpful becuase it didnt tell me which one I messed up. This simple patch adds the name to the error messages.

### CODE REVIEW

File: master/buildbot/schedulers/forcesched.py [hunk 1]
```diff
@@ -542,30 +542,30 @@ def __init__(self, name, builderNames,
                          name)
 
         if not self.checkIfListOfType(builderNames, str):
-            config.error("ForceScheduler builderNames must be a list of strings: %r" %
-                         builderNames)
+            config.error("ForceScheduler '%s' builderNames must be a list of strings: %r" %
```
Comment (sa2ajj) [inline master/buildbot/schedulers/forcesched.py]: [nit] maybe add `:` after `'%s'`? for consistency

**Comment by sa2ajj** [comment]
Beside the nits, LGTM

File: master/buildbot/schedulers/forcesched.py (hunk 1 above)
Comment (jaredgrubb) [inline master/buildbot/schedulers/forcesched.py]: Sure, I suppose that makes it easier to read. Will do.

**Comment by jaredgrubb** [comment]
Added the colon.

**Pull request merged**